contracts/SummitVRFModule.sol

```diff
@@ -0,0 +1,185 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.0;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+
+/*
+---------------------------------------------------------------------------------------------
+--   S U M M I T . D E F I
+---------------------------------------------------------------------------------------------
+
+
+Summit is highly experimental.
+It has been crafted to bring a new flavor to the defi world.
+We hope you enjoy the Summit.defi experience.
+If you find any bugs in these contracts, please claim the bounty (see docs)
+
+
+Created with love by Architect and the Summit team
+
+
+
+
+
+---------------------------------------------------------------------------------------------
+--   C U S T O M V R F
+---------------------------------------------------------------------------------------------
+
+
+It is also responsible for requesting and receiving the trusted seeds, as well as their decrypted versions after the future block is mined
+
+
+RANDOM NUMBER GENERATION
+    . Webservice queries `nextSeedRoundAvailable` every 5 seconds
+    . When true received
+    . Webservice creates a random seed, seals it with the trustedSeeder address, and sends it to `receiveSealedSeed`
+    . When the futureBlockNumber set in `receiveSealedSeed` is mined, `futureBlockMined` will return true
+    . Webservice sends the original unsealed seed to `receiveUnsealedSeed`
+*/
+
+contract SummitVRFModule is Ownable {
```

Comment:
This needs to inherit from `ISummitVRFModule` to ensure that the functions are implemented correctly

Reply:
sure. i will fix it soon.

Reply:
just fixed